src/common/transformations/include/transformations/common_optimizations/nop_elimination.hpp

```diff
@@ -83,6 +84,16 @@ class ov::pass::EliminateConcat : public ov::pass::MatcherPass {
     EliminateConcat();
 };
 
+/**
+ * @ingroup ov_transformation_common_api
+ * @brief EliminateConcatSplit eliminates split from concat that no need
```

Comment:
Could you please clarify what that means
"split from concat that no need"?
please add more details to the description.
As I can see from the code, we try to detect Concat->StridedSlice patterns, should we change the transformation name accordingly? Or point that the StridedSlices behavior is equal to Split, please add an example to the desc